From 03c3df5e36bc59c1cf67d1103b691c92e4272965 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Tue, 2 Jun 2015 16:51:45 +0200
Subject: [PATCH] build-sys: Always prepend '-' to BUILDID

This was removed by commit 28a6bd6 as WINDOWS_PRODUCTVERSION
needs a buildid without a dash. Apart from this variable,
all other uses of buildid/BUILDID in virt-viewer source tree
need a dash between the version number and the buildid to avoid getting
output like "3.01" instead of "3.0-1"

Rather than patching every location where BUILDID is used, this commit
appends the "-" before substituting/defining BUILDID in configure.ac.
This does not modifies the buildid configure.ac variable, this way
WINDOWS_PRODUCTVERSION won't get an unwanted '-'.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index d6a438c..d893631 100644
--- a/configure.ac
+++ b/configure.ac
@@ -224,8 +224,8 @@ AC_ARG_WITH([buildid],
     [buildid="$with_buildid"], [buildid="0"])
 
 if test "x$buildid" != "x0"; then
-    AC_DEFINE_UNQUOTED([BUILDID], "$buildid", [Build version details])
-    AC_SUBST([BUILDID], "$buildid")
+    AC_DEFINE_UNQUOTED([BUILDID], "-$buildid", [Build version details])
+    AC_SUBST([BUILDID], "-$buildid")
 else
     AC_DEFINE_UNQUOTED([BUILDID], "", [Build version details])
 fi
